content/docs/fairing/azure/az-kubeflow-notebook.md

```diff
@@ -0,0 +1,186 @@
++++
+title = "Train and Deploy on Azure from a Kubeflow Notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Azure Kubernetes Service (AKS) from a notebook that is hosted on Kubeflow"
+weight = 45
++++
+
+This page documents how to run the [Fairing prediction example
+notebook][xgb-notebook] on [Azure Kubernetes Service
+(AKS)][az-kubernetes] in a notebook hosted on Kubeflow.
+
+## Prerequisites
+
+Before you configure and use the Kubeflow Fairing Python SDK, you must have a
+Kubeflow environment installed. The example notebook does in-cluster training.
+The example notebook has been tested to work in a cluster with 2 nodes and
+virtual machines with Standard_D4_v3 size. It is not recommended to use the
+smallest size virtual machines.
+
+* If you do not have a Kubeflow installation on Azure, follow the [installation
+  guide][kf-az-deploy].
+* You must have the `kubectl` command line interface installed and configured
+  to use the Kubernetes cluster where Kubeflow is installed. In the preceding
+  installation guide, the command `az aks get-credentials` configures `kubectl`
+  to access your Kubernetes cluster.
+
+## Create Azure Container Registry and Storage
+
+In this example notebook, Azure Container Registry (ACR) is used to host docker
+images for deployment and Azure Storage (Storage) is used as build context for
+in-cluster building.
+
+You can re-use existing ACR and Storage or create new ones. For more
+information, see the documentation for [ACR][az-container-reg] and
+[Storage][az-storage].
+
+After you have created your ACR and Storage resources, you must configure a
+Service Principal with access to these resources. If you want to use the same
```

Comment:
I think we can remove the capital letters on "service principal" here and below. The referenced [Microsoft docs](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal) don't capitalize the term.

Reply:
Thanks for catching that!